### PR TITLE
Adds a set of admin stock parts intended for debugging

### PR DESCRIPTION
**Pull request by genessee596**
## About The Pull Request

Adds a set of stock parts with 100 efficiency, which is roughly 95 times better than the best stock parts the crew can get (one star parts). These decrease the price of printing things significantly - by 80%, specifically. And, you know, increase the efficiency of anything else that uses stock parts. I.E. everything.

Due to limitations in the codebase, they can't really get anymore efficient than 100. Trust me, I tried 1000 and nothing changed.

Uses unused sprites already existing in the codebase - .dmi file just renames them from 'unused' to their respective names.

## Why It's Good For The Game

Helps coders test things such as autolathes and their subtypes, design disk costs, and pretty much anything else that uses stock parts

## Changelog
:cl:
add: Admin stock parts intended for debugging
/:cl:

### CODE REVIEW

**Review by Evankhell561 — COMMENTED** [review]
is there something like a rapid part exchange device in our repository? In my experience with the stock parts used for debugging they are not very useful if they do not all come together in an RPED.

File: code/game/objects/items/weapons/stock_parts.dm [hunk 1]
```diff
@@ -311,6 +311,53 @@
...
+	spawn_blacklisted = TRUE
+
+/obj/item/weapon/stock_parts/matter_bin/debug
+	name = "bluespace matter bin"
+	desc = "An ultra-advanced container that opens into a localized pocket dimension meant for holding compressed matter. Used in construction of certain devices."
+	icon_state = "debug_matter_bin"
+	origin_tech = list(TECH_MATERIAL = 6)
+	rating = 100
+	matter = list(MATERIAL_PLASTIC = 3, MATERIAL_GLASS = 1, MATERIAL_PLASMA = 1)
+	spawn_blacklisted = TRUE
```
Comment (Evankhell561) [inline code/game/objects/items/weapons/stock_parts.dm:360]: I'd rather you use bad_type or spawn_frequency = 0

**Review by Evankhell561 — DISMISSED** [review]

File: code/game/objects/items/weapons/stock_parts.dm (hunk 1 above)
Comment (genessee596) [reply code/game/objects/items/weapons/stock_parts.dm:360]: hokay

**Comment by genessee596** [comment]
ok done

**Review by TheShown911 — APPROVED** [review]

**Review by TorinoFermic — APPROVED** [review]

**Pull request merged**